sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeDirectoryClient.cs

```diff
@@ -158,6 +158,53 @@ public DataLakeDirectoryClient(Uri directoryUri, StorageSharedKeyCredential cred
         {
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DataLakeDirectoryClient"/>
+        /// class.
+        /// </summary>
+        /// <param name="directoryUri">
+        /// A <see cref="Uri"/> referencing the directory that includes the
+        /// name of the account, the name of the file system, and the path of the
+        /// directory.
+        /// Must not contain shared access signature.
```

Comment:
I would add ", which should be passed in the second (following) parameter" ... or something like that.

Reply:
done.